sn_node/Cargo.toml

```
@@ -55,7 +60,7 @@ rand-07 = { package = "rand", version = "~0.7.3" }
 rayon = "1.5.1"
 rmp-serde = "1.0.0"
 self_encryption = "~0.27.5"
-sn_consensus = "3.2.1"
+sn_consensus = { version = "3.2", git = "https://github.com/maidsafe/sn_consensus" }
```

ditto, drop the git ref
```suggestion
sn_consensus = "3.2.1"
```